src/context/nft_profile.tsx

```diff
@@ -51,6 +57,37 @@ export const NFTProfileProvider: FC<{ children: ReactNode }> = ({ children }) =>
     })
   }
 
+  const fetchNFTDetails = useCallback(async (nftData: any): Promise<INFTMetadata[]> => {
```

Comment:
Made new functions in the nft_profile context which we can use and remove mostly the request in the ContentProfile but wanted to be sure if we would use the external nfts pull anywhere else if not, there won't be need for this and I'll remove it